### PR TITLE
Style core elements

### PR DESCRIPTION
**Pull request by herokudev**
Add styling to the application to make it match this wire-frame.

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -17,21 +16,19 @@ class Library {
...
+  static addBookToList(book) {
+    const row = document.createElement('tr');
+    row.innerHTML = `
+    <td>"${book.title}" by ${book.author} </td>
+    <td><a href="#" class="btn btn-danger btn-sm delete text-center remove-book ms-auto bg-white border border-2 border-dark text-dark px-1 d-flex align-items-end">Remove</a></td>    
+    `;
+    booksList.appendChild(row);
+  }
+
   addBook() {
```
Comment (rloterh) [inline main.js:32]: - [ ] OPTIONAL 
Your `addBook`  function is not working as expected. When the `add` button is clicked a new book is supposed to be added to the collection, with **title** and **author**, however when just the _book title_  or the _ book author_ alone is entered your app accepts the data. Kindly improve upon this aspect by ensuring that both _book title_ and _book author_ are entered before data is accepted.

File: index.html [hunk 2]
```diff
@@ -4,31 +4,39 @@
...
+        </div>
+        <div class="addBook-container">
+            <div>
+                <input type="text" id="addTitle" class="book_title" placeholder="Title" required><br>
+            </div>
+            <div>
+                <input type="text" id="addAuthor" class="book_author" placeholder="Author" required><br>
+            </div>
+            <div class="bnt-position">
+                <input type="submit" id="addBook" class="addButton" value="Add">
```
Comment (rloterh) [inline index.html:36]: - [ ] Your add button is not properly aligned. From the provided [wireframe design](https://github.com/microverseinc/curriculum-javascript/blob/bb245ca8a9d3bb04ebefbbd45e800764e1fc5f74/books/images/awesome_books_core_elements.png) we can see that the _add button_  is aligned to the right of the input fields. Kindly note that we actually wanted the design to as much as possible follow the provided [wireframe design](https://github.com/microverseinc/curriculum-javascript/blob/bb245ca8a9d3bb04ebefbbd45e800764e1fc5f74/books/images/awesome_books_core_elements.png). Kindly ensure to improve upons this aspect of your design.

**Your desgin:**
image

**The wireframe design:**
image

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Kingsley  & Herbert, 

You did extremely great on this project!👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there!👍 

## Good Points 👍  
- No linter errors


## Required Changes ♻️
- [ ] _Kindly check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- [ ] _Kindly check the comments marked **[OPTIONAL]** under the review._

<br>
Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by bellom — APPROVED** [review]
## Hi @herokudev  👏  

**Great work making all the necessary corrections from the previous review.**

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

### Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**